examples/custom_route.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from aiohttp import web
+import typing
+
+from aiogram import Bot, Dispatcher, executor, types
+from aiogram.dispatcher.webhook import WebhookRequestHandler
+from aiogram.utils.executor import Executor
+from loguru import logger
+
+
+API_TOKEN = 'BOT TOKEN'
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
why did you import `loguru` above?

Reply:
it was only for test purpose. You right i should remove this line it may confuse.